editor/tree_search.cpp

```diff
@@ -95,6 +93,9 @@ void TreeSearchPanel::_notification(int p_what) {
 			line_edit_search->connect("text_submitted", c_text_submitted.unbind(1));
 			break;
 		}
+		case NOTIFICATION_THEME_CHANGED: {
+			label_filter->set_text(TTR("Filter"));
+		}
```

Comment:
Missing `break`. In Godot, there is a convention to use such idiom:
```cpp
case X: {
} break;
```
I think we should adopt it as well.

Reply:
Done.